### PR TITLE
[Hub Generated] Review request for Microsoft.CostManagement to add version stable/2019-10-01

### PR DESCRIPTION
**Pull request by shbha1**
If you are a MSFT employee you can view your [work branch via this link](https://portal.azure-devex-tools.com/branch/shbha1/azure-rest-api-specs/dev-cost-management-Microsoft.CostManagement-2019-10-01...master/).

### Contribution checklist:
- [ ] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

**Review by yungezz — APPROVED** [review]

**Comment by yungezz** [comment]
hi @shbha1 could you pls fix CI failure, then the PR ok to merge.

**Comment by yungezz** [comment]
ping for update

**Comment by shbha1** [comment]
Please confirm if we can merge without fixing the code-style issue the recent check is complaining about.

**Comment by shbha1** [comment]
What is pending to get this PR merged?

**Comment by yungezz** [comment]
hi @ArcturusZhang  could you pls help to look at the GO SDK generation failure? thanks. https://dev.azure.com/azure-sdk/internal/_build/results?buildId=217896&view=logs&j=fae805ed-8bde-5aa7-6387-2396f34533f9&t=c2e6587e-d03c-5fa9-0378-bc3c21e64d38

**Comment by ArcturusZhang** [comment]
> hi @ArcturusZhang could you pls help to look at the GO SDK generation failure? thanks. https://dev.azure.com/azure-sdk/internal/_build/results?buildId=217896&view=logs&j=fae805ed-8bde-5aa7-6387-2396f34533f9&t=c2e6587e-d03c-5fa9-0378-bc3c21e64d38

One example file is malformed. I wonder why other CI does not report this.

File: specification/cost-management/resource-manager/Microsoft.CostManagement/stable/2019-10-01/examples/ExternalSubscriptionsDimensions.json [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+						"description": "Resource ID",
+						"usageStart": "2019-12-01T00:00:00Z",
+						"usageEnd": "2019-12-31T00:00:00Z",
+						"filterEnabled": true,
+						"groupingEnabled": true
+					}
+				}]
+			}
+		}
+	}
```
Comment (ArcturusZhang) [inline specification/cost-management/resource-manager/Microsoft.CostManagement/stable/2019-10-01/examples/ExternalSubscriptionsDimensions.json]: Missing one close curly brace here, if you count carefully.

**Review by ArcturusZhang — CHANGES_REQUESTED** [review]

**Comment by ArcturusZhang** [comment]
And there are also a lot of field that is reported as
```
model/property lacks 'description' and 'title' property. Consider adding a 'description'/'title' element. Accurate description/title is essential for maintaining reference documentation.
```
I highly recommend you to add description or title for them. If it is by design, just ignore this.
And autorest also reports some other warnings like this:
```
OperationId should contain the verb: 'query' in:'Query_Usage'. Consider updating the operationId
```
Conventionally, `operationId` should be a `puralNoun_Verb` form.

**Comment by yungezz** [comment]
> > hi @ArcturusZhang could you pls help to look at the GO SDK generation failure? thanks. https://dev.azure.com/azure-sdk/internal/_build/results?buildId=217896&view=logs&j=fae805ed-8bde-5aa7-6387-2396f34533f9&t=c2e6587e-d03c-5fa9-0378-bc3c21e64d38
> 
> One example file is malformed. I wonder why other CI does not report this.

hi @raych1 could you pls help to have a look why malformed example isn't reported? Thanks.

**Comment by raych1** [comment]
> > > hi @ArcturusZhang could you pls help to look at the GO SDK generation failure? thanks. https://dev.azure.com/azure-sdk/internal/_build/results?buildId=217896&view=logs&j=fae805ed-8bde-5aa7-6387-2396f34533f9&t=c2e6587e-d03c-5fa9-0378-bc3c21e64d38
> > 
> > 
> > One example file is malformed. I wonder why other CI does not report this.
> 
> hi @raych1 could you pls help to have a look why malformed example isn't reported? Thanks.

@yungezz , model validation works as expected which has the JSON format error reported on the last 2nd commit CI checks [result](https://dev.azure.com/azure-sdk/public/_build/results?buildId=216252&view=logs&jobId=8d2f9c49-cd83-5a9d-f3fe-2fd651afa742&j=8d2f9c49-cd83-5a9d-f3fe-2fd651afa742&t=43324f1f-6724-5d61-785f-f0ac9fd256a1).

File: specification/cost-management/resource-manager/Microsoft.CostManagement/stable/2019-10-01/examples/ExternalSubscriptionsDimensions.json (hunk 1 above)
Comment (shbha1) [reply specification/cost-management/resource-manager/Microsoft.CostManagement/stable/2019-10-01/examples/ExternalSubscriptionsDimensions.json]: Thank you for pointing this out, I've added the missing curly bracket

**Comment by shbha1** [comment]
can we please merge the PR (the failing test says 'prettier check' - not required check failed).

**Comment by raych1** [comment]
It's interesting that OAV doesn't report when the missing curly bracket is in the end. I logged an issue to track this.

**Review by raych1 — APPROVED** [review]

**Comment by raych1** [comment]
@ArcturusZhang , you requested the changes, can you please review this again?

**Review by ArcturusZhang — APPROVED** [review]

**Pull request merged**